### PR TITLE
feat(log): colors for console logs

### PR DESCRIPTION
**Pull request by geman220**
### Description
- [Example](https://i.ibb.co/h9yvKyg/Untitled.png) colors
- Updated `.env-example` to eliviate continual questions and support in discord and github Issues.  There are probably 4-5 `pushover` configuration problems in #help every hour.  Users are also constantly asking "What does ______ error mean?" and it's almost always due to users keeping all defaults enabled in `.env`

### New dependencies
- Colors 1.4.0

### CODE REVIEW

**Comment by geman220** [comment]
Is there some way I should be doing this aside from using `+`?  Works fine on my end.

**Comment by ethanspitz** [comment]
I think this was just rendered obsolute (the logic updates at least) by #141

**Comment by geman220** [comment]
Removed newegg logic

**Comment by geman220** [comment]
Resolved conflicts.

**Comment by geman220** [comment]
Resolved and moved `colors` to `logger.ts`

File: .env-example [hunk 1]
```diff
@@ -1,33 +1,79 @@
-BROWSER_TRUSTED="false"
-DISCORD_NOTIFY_GROUP="@here"
-DISCORD_WEB_HOOK="https://discordapp.com/api/webhooks/23123123123/5dfsdfh3h2j5hdfhsdfsdhj55jf"
-EMAIL_USERNAME="youremail@gmail.com"
-EMAIL_PASSWORD="secretpassword"
-HEADLESS="true"
-IN_STOCK_WAIT_TIME="30"
-LOG_LEVEL="info"
+// All configuration variables are optional.  Please read https://github.com/jef/nvidia-snatcher#customization for help on customizing this file.
```
Comment (jef) [inline .env-example:1]: I don't mind this top comment, but do you mind reverting the rest? I feel like it'll be hectic keeping up with doc here and `README` and everywhere else.

Comment (geman220) [reply .env-example:1]: All set

**Review by jef — APPROVED** [review]

**Pull request merged**